modules/core/core-api/src/main/java/com/enonic/xp/project/Projects.java

```diff
@@ -39,6 +39,14 @@ public static Projects from( Repositories repositories )
             build();
     }
 
+    public Project get( final ProjectName name )
+    {
+        return this.stream().
+            filter( project -> project.getName().equals( name ) ).
+            findAny().
```

Comment:
Change to findFirst() so when we support multi-inheritance first project will be returned